examples/jsonpath-expressions/jsonpath_expressions.md

```diff
@@ -0,0 +1,11 @@
+# JSONPath expressions
+
+JSONPath expressions provide a straightforward method for navigating the structure of JSON data.
```

Comment:
Elaborate a bit to say Ballerina supports evaluating JSONPath expressions against `json` values via the `ballerina/data.jsondata` module.